docs/src/domains/cdn/fastly.md

```diff
@@ -4,7 +4,7 @@ weight: 1
 description: |
   In some cases you may want to opt to use a CDN such as Fastly rather than the Platform.sh router's cache.  Using a CDN can offer a better time-to-first-byte for cached content across a wider geographic region at the cost of the CDN service.
 aliases:
-  - "/golive/steps/fastly.html"
+  - "/domains/in-depth/fastly.html"
```

Comment:
```suggestion
  - "/golive/steps/fastly.html"
```

Reply:
This should stay the same - it's essentially the built-in Hugo redirect option for the old location.